internal/key/provider.go

```diff
@@ -41,3 +41,10 @@ func PureCAPIProviders() []string {
 		ProviderCloudDirector,
 	}
 }
+
+// CAPIProvidersThatUseReleases is the list of CAPI providers which are using Release resources.
+func CAPIProvidersThatUseReleases() []string {
```

Comment:
same here maybe `ProviderUsingReleases` or `CAPIProviderUsingReleases`

Reply:
yup, good point, will change these

Reply:
✅ renamed it